org.lflang/src/org/lflang/generator/c/CTriggerObjectsGenerator.java

```diff
@@ -161,6 +166,13 @@ public static String generateInitializeTriggerObjects(
             isFederated,
             clockSyncIsOn
         ));
+        if (targetConfig.schedulerType == SchedulerOption.QS) {
```

Comment:
Why not have this inside `generateSchedulerInitializer`?

Reply:
Right, it makes sense to have this in `generateSchedulerInitializer`. Thanks for the catch!